src/main/java/com/greenfox/tribes1/Security/WebSecurity.java

```diff
@@ -22,14 +22,12 @@ public WebSecurity(UserDetailsServiceImpl userDetailsService) {
   @Override
   public void configure(HttpSecurity http) throws Exception {
     http.csrf().disable().authorizeRequests()
-            .antMatchers( "/register", "/login").permitAll()
+            .antMatchers( "/register", "/login", "/test", "/heroku").permitAll()
```

Comment:
In this situation, you can easily use "/**"

Reply:
Not wanted to add permit to all endpoints with /** in oder to not mess the code up later on ...

Reply:
corrected as per requested